Library/Formula/coreutils.rb

```diff
@@ -31,6 +31,16 @@ class Coreutils < Formula
   depends_on "gmp" => :optional
 
   def install
+    # Work around unremovable, nested dirs bug that affects lots of
+    # GNU projects. See:
+    # https://github.com/Homebrew/homebrew/issues/45273
+    # https://github.com/Homebrew/homebrew/issues/44993
+    # This is thought to be an el_captian bug:
+    # http://lists.gnu.org/archive/html/bug-tar/2015-10/msg00017.html
+    if MacOS::Version == :el_capitan
```

Comment:
You're right.  I don't know why I did that.  I've never done that before.  I'll fix it!